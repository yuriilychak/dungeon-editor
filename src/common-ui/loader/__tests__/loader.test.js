import React from "react";
import { createShallow } from "@material-ui/core/test-utils";
import Loader from "../loader";

describe("Loader test",()=> {

    it('default snapshot', () => {
        expect(createShallow()(
            <Loader
                size={30}
                isLoading
            >
                Test
            </Loader>
        ).html()).toMatchSnapshot();
    });

    it('default snapshot with loading disabled', () => {
        expect(createShallow()(
            <Loader
                size={30}
                isLoading={false}
            >
                Test
            </Loader>
        ).html()).toMatchSnapshot();
    });
});
