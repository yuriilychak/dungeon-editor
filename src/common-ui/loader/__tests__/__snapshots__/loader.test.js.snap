// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader test default snapshot 1`] = `"<div><div class=\\"MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate\\" style=\\"width:30px;height:30px\\" role=\\"progressbar\\"><svg class=\\"MuiCircularProgress-svg\\" viewBox=\\"22 22 44 44\\"><circle class=\\"MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate\\" cx=\\"44\\" cy=\\"44\\" r=\\"20.2\\" fill=\\"none\\" stroke-width=\\"3.6\\"></circle></svg></div></div>"`;

exports[`Loader test default snapshot with loading disabled 1`] = `"Test"`;
